influxdb_client/client/flux_csv_parser.py

```diff
@@ -256,3 +263,27 @@ def add_column_names_and_tags(table, csv):
     def _insert_table(self, table, table_index):
         if self._serialization_mode is FluxSerializationMode.tables:
             self.tables.insert(table_index, table)
+
+    def _is_profiler_record(self, flux_record: FluxRecord) -> bool:
+        if self._profilers is None:
+            return False
+
+        for profiler in self._profilers:
+            if flux_record.get_measurement() == "profiler/" + profiler:
```

Comment:
This will not work if the query response doesn't contains `_measurement`. Try to use query: `buckets()` or run https://github.com/influxdata/influxdb-client-python/blob/feat/profiler_query/examples/query.py with enabled profiling.